src/screens/Shared/ViewBranchMachine/components/TabPaymentsReceived/index.tsx

```diff
@@ -0,0 +1,246 @@
+			const receiptsData = collectionReceiptsData.list.map((receipt) => ({
+				key: `receipt-${receipt.id}`,
+				datetime: formatDateTime(receipt.datetime_created),
+				receipt_type: 'Collection Receipt',
+				invoice: receipt.order_of_payment ? (
+					<Button
+						className="pa-0"
+						type="link"
+						onClick={() => setSelectedCollectionReceipt(receipt)}
+					>
+						{receipt.order_of_payment.id}
+					</Button>
+				) : (
+					'N/A'
+				),
+				payment: formatInPeso(receipt.amount),
+				cashier: getFullName(receipt.created_by),
+				mode_of_payment: modeOfPayment[receipt.mode],
+				remarks: `OP: ${receipt.order_of_payment.id}`,
+			}));
+
+			// Merge transactions and receipts
+			const mergedData = [...transactionsData, ...receiptsData];
+
+			// Sort the merged data by datetime to mix transactions and receipts
+			const sortedData = mergedData.sort((a, b) =>
+				a.datetime.localeCompare(b.datetime, undefined, { numeric: true }),
+			);
+
+			setDataSource(sortedData);
+		}
+	}, [transactions, collectionReceiptsData]);
+
+	return (
+		<>
+			<TableHeader title="Payments Received" wrapperClassName="pt-2 px-0" />
+
+			<RequestErrors
+				errors={
+					convertIntoArray(transactionsError) &&
+					convertIntoArray(collectionReceiptsError)
+				}
+			/>
+
+			<Filter
+				isLoading={
+					isFetchingTransactions &&
+					!isTransactionsFetchedAfterMount &&
+					isFetchingCollectionReceipts
+				}
+			/>
+
+			{voidedStatuses.includes(_.toString(params?.statuses)) && (
+				<TransactionsCancelled
+					statuses={_.toString(params?.statuses)}
+					timeRange={_.toString(params?.timeRange)}
+				/>
+			)}
+
+			<Summary branchMachineId={branchMachineId} />
+
+			<Table
+				columns={columns}
+				dataSource={dataSource}
+				loading={isFetchingTransactions && !isTransactionsFetchedAfterMount}
+				pagination={{
+					current: Number(params.page) || DEFAULT_PAGE,
+					total,
+					pageSize: Number(params.pageSize) || DEFAULT_PAGE_SIZE,
+					onChange: (page, newPageSize) => {
+						setQueryParams({
+							page,
+							pageSize: newPageSize,
+						});
+					},
+					disabled: !dataSource,
+					position: ['bottomCenter'],
+					pageSizeOptions,
+				}}
+				scroll={{ x: 800 }}
+				bordered
+			/>
+
+			{selectedTransaction && (
+				<ViewTransactionModal
+					siteSettings={siteSettings}
+					transaction={selectedTransaction}
+					onClose={() => setSelectedTransaction(false)}
+				/>
+			)}
+			{selectedCollectionReceipt && siteSettings && (
+				<ViewCollectionReceiptModal
+					collectionReceipt={selectedCollectionReceipt}
+					siteSettings={siteSettings}
+					onClose={() => setSelectedCollectionReceipt(null)}
+				/>
+			)}
+		</>
+	);
+};
+
+interface FilterProps {
+	isLoading: boolean;
+}
+
+const Filter = ({ isLoading }: FilterProps) => {
+	return (
+		<Row className="mb-4" gutter={[16, 16]}>
+			<Col lg={12} span={24}>
+				<TimeRangeFilter disabled={isLoading} />
+			</Col>
+		</Row>
+	);
+};
```

Comment:
Remove `return` keyword.

Reply:
This is needed.

'Filter' cannot be used as a JSX component.
  Its return type 'void' is not a valid JSX element.

Reply:
This is what I meant: 
![image](https://github.com/user-attachments/assets/1347cdbf-592b-45a9-8cbd-731915c9b87b)